### DISCUSSION TITLE
i cant run

### DISCUSSION THREAD

**Discussion by orkaroeli** [curl/curl#15189] [General]
the configure command is not found on debian without root access or passwords...just i can access desktop with given from admin password. i trying to install curl i have no acccess to apt or apt-get what i do to install curl? various commands like make does not installed

**Comment by dfandrich** [comment]
The easiest way to get a curl on such a system is to install a static binary
rather than trying to compile it yourself.  There are a number of options
available at https://curl.se/download.html#Linux, such as all the Travis
options listed there.

**Reply by orkaroeli** [reply]
But i need to use make install or some make command? To install? Is the tar.gz file listed?

**Comment by dfandrich** [comment]
If you install a binary, it's already compiled so you just need to extract it and run it.

**Comment by vszakats** [comment]
You can find daily binaries for Linux on this link too:
https://github.com/curl/curl-for-win/actions/workflows/daily.yml
Then download the 'curl-linux-musl-daily-tool' package.
